main.go

```diff
@@ -137,6 +140,12 @@ func coverage(coverpkg, pkg, covermode, cpu, parallel, timeout string, short, v
 	if v {
 		args = append(args, "-v")
 	}
+	if race {
+		if covermode != "" && covermode != "atomic" {
+			log.Fatalf("cannot use race flag and covermode=%s. See more detail on golang/go#12118.", covermode)
```

Comment:
That's exactly what I expected! ... except we should check "race" flag too ;)

And, please just return "err".